ResidentsSocialCarePlatformApi/V1/Boundary/Responses/CaseNoteInformation.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+
+namespace ResidentsSocialCarePlatformApi.V1.Boundary.Responses
+{
+    public class CaseNoteInformation
+    {
+        public string MosaicId { get; set; }
+
+        public int CaseNoteId { get; set; }
+
+        public int PersonVisitId { get; set; }
+
+        public string NoteType { get; set; }
+
+        public string CaseNoteTitle { get; set; }
+
+        public DateTime EffectiveDate { get; set; }
+
+        public DateTime CreatedOn { get; set; }
+
+        public DateTime LastUpdatedOn { get; set; }
+
+        public string CreatedByName { get; set; }
+
+        public string CreatedByEmail { get; set; }
+
+        public string LastUpdatedName { get; set; }
+
+        public string LastUpdatedEmail { get; set; }
+
+        public string CaseNoteContent { get; set; }
+
+        public int RootCaseNoteId { get; set; }
+
+        public DateTime CompletedDate { get; set; }
+
+        public DateTime TimeoutDate { get; set; }
+
+        public int CopyOfCaseNoteId { get; set; }
```

Comment:
In terms of `CopyOfCaseNoteId` might we also include the case notes that were created as copies of this one? And related people or mosaic ids? Might be something to add later of course.

Reply:
👍🏾